src/utils/buildAxis.linear.ts

```diff
@@ -117,24 +118,25 @@ function buildTimeAxis<TDatum>(
     return value
   })
 
-  let shouldNice = true
+  let shouldNice = options.shouldNice
```

Comment:
This `let` is probably unnecessary now. We can just reference the `options.shouldNice` :)

Reply:
I left it there intentionally so that even if it's set to `true`, it gets overridden in case `min`/`max`/`hardMin`/`hardMax` is provided?

Reply:
@tannerlinsley is this what you meant? Though I'm not sure how I feel about mutating function arguments :)

Reply:
Oh, oh, I forgot that we change it. the `let` is fine then. Sorry!